ambry-server/src/integration-test/java/com.github.ambry.server/ServerTestUtil.java

```diff
@@ -516,584 +516,550 @@ protected static void endToEndReplicationWithMultiNodeMultiPartitionTest(int int
       Port dataNode1Port, Port dataNode2Port, Port dataNode3Port, MockCluster cluster, SSLConfig clientSSLConfig1,
       SSLConfig clientSSLConfig2, SSLConfig clientSSLConfig3, SSLSocketFactory clientSSLSocketFactory1,
       SSLSocketFactory clientSSLSocketFactory2, SSLSocketFactory clientSSLSocketFactory3,
-      MockNotificationSystem notificationSystem, boolean testEncryption) {
+      MockNotificationSystem notificationSystem, boolean testEncryption) throws Exception {
```

Comment:
The changes in this file can be ignored (just removed a try-catch to let the exception be thrown out of the method).